example/App.tsx

```diff
@@ -98,6 +98,17 @@ function useIsFabricComponent() {
   return [isFabric, setter] as const;
 }
 
+function useLocalStorageStatus() {
+  const [localValue, setLocalValue] = useState("Checking");
+  useEffect(() => {
+    const key = "sample/local-storage";
+    window?.localStorage?.setItem(key, "Available");
+    setLocalValue(window?.localStorage?.getItem(key) ?? "Error");
+    return () => window?.localStorage?.removeItem(key);
+  }, []);
+  return localValue;
+}
```

Comment:
ngl it kind of got shivers down my spine for a second seeing this web code - damn 🔥